client/me/purchases/manage-purchase/change-payment-method/index.jsx

```diff
@@ -446,6 +466,11 @@ function useAssignablePaymentMethods() {
 		activePayButtonText: translate( 'Save card' ),
 	} );
 
+	const payPalMethod = useCreatePayPal( {
+		labelText:
+			currentPaymentMethodId === 'paypal-existing' ? translate( 'New PayPal account' ) : 'PayPal',
```

Comment:
Should we translate "PayPal" here too?  I'm unclear if it's actually supposed to be translated (since it's a proper name) but it looks like some other places translate it already, and if nothing else doing so gives translators maximum freedom.

Reply:
I intentionally didn't, but I don't mind doing so.